src/chart.ts

```diff
@@ -22,6 +22,14 @@ export interface ChartProps {
    */
   readonly labels?: { [name: string]: string };
 
+  /**
+   * The autogenerated resource name by default is suffixed with a stable hash
+   * of the construct path. Setting this property to false drops the hash suffix.
+   *
+   * @default true
+   */
+  readonly enableResourceNameHashes?: boolean;
```

Comment:
Normally to activate a feature, we set a certain property to `true`, not `false`. The feature we want to activate here is actually to *disable* hashes...So I would suggest:

```suggestion
  readonly disableResourceNameHashes?: boolean;
```

And set the default to `false`. This would allow to set `disableResourceNameHashes: true`, which conveys the user intent more clearly I think.